src/pages/BlendPoolPage.tsx

```diff
@@ -39,6 +39,15 @@ type PoolParams = {
   pooladdress: string;
 };
 
+export type AccountData = {
+  address: string;
+  connector: Connector<any, any> | undefined;
+    ens: {
+        avatar: string | null | undefined;
+        name: string;
+    } | undefined;
+}
+
```

Comment:
Is this exported by `wagmi` anywhere? Ideally we shouldn't need to redefine it here.

Reply:
Actually I looked through the `wagmi` 0.4.x docs and it looks like the type declaration is different in that the ENS is not explicitly included in the account data. Since we're not using the ENS anywhere from this type declaration, could we drop this declaration and use the `wagmi` one instead?

Reply:
Could you point me to the type definition in the docs? I can't seem to find it

Reply:
Neither `wagmi` 0.4.x nor 0.5.x export the `GetAccountResult` type that is returned by the `getAccount` action (the one used by `useAccount` hook). To be more precise, they are exported from `index.ts` but don't make their way to the npm package. Unclear if this is due to misconfigured `package.json` or the fact that `GetAccountResult` is dependent on a generic type constraint that _doesn't_ get exported... but regardless Ian's solution is the best one here